docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md

```diff
@@ -183,17 +183,17 @@ let fullContext = context.DataContext
 let nullable value = new System.Nullable<_>(value)
 
 let addInstructor(lastName, firstName, hireDate, office) =
-let hireDate = DateTime.Parse(hireDate)
-let newPerson = new EntityConnection.ServiceTypes.Person(LastName = lastName,
+  let hireDate = DateTime.Parse(hireDate)
+  let newPerson = new EntityConnection.ServiceTypes.Person(LastName = lastName,
                                                          FirstName = firstName,
                                                          HireDate = nullable hireDate)
```

Comment:
Nit: I think these two lines should be indented too, so that they stay aligned with the first argument.